src/main/java/net/william278/velocitab/packet/PacketRegistration.java

```diff
@@ -72,16 +78,48 @@ public PacketRegistration<P> mapping(
         }
 
         public void register() {
+
             try {
                 final StateRegistry.PacketRegistry packetRegistry = direction == ProtocolUtils.Direction.CLIENTBOUND
                         ? (StateRegistry.PacketRegistry) STATE_REGISTRY$clientBound.invoke(stateRegistry)
                         : (StateRegistry.PacketRegistry) STATE_REGISTRY$serverBound.invoke(stateRegistry);
+
                 PACKET_REGISTRY$register.invoke(
                         packetRegistry,
                         packetClass,
                         packetSupplier,
                         mappings.toArray(StateRegistry.PacketMapping[]::new)
                 );
+
+            } catch (Throwable t) {
+                throw new RuntimeException(t);
+            }
+        }
+
+        @SuppressWarnings("unchecked")
+        public void unregister() {
+            try {
+
+                final StateRegistry.PacketRegistry packetRegistry = direction == ProtocolUtils.Direction.CLIENTBOUND
+                        ? (StateRegistry.PacketRegistry) STATE_REGISTRY$clientBound.invoke(stateRegistry)
+                        : (StateRegistry.PacketRegistry) STATE_REGISTRY$serverBound.invoke(stateRegistry);
```

Comment:
```suggestion
            try {
                final StateRegistry.PacketRegistry packetRegistry = direction == ProtocolUtils.Direction.CLIENTBOUND
                        ? (StateRegistry.PacketRegistry) STATE_REGISTRY$clientBound.invoke(stateRegistry)
                        : (StateRegistry.PacketRegistry) STATE_REGISTRY$serverBound.invoke(stateRegistry);
```
style nit: just removing this newline here